platforms/documentation/docs/src/docs/userguide/authoring-builds/basics/plugins.adoc

```diff
@@ -407,8 +420,31 @@ include::sample[dir="snippets/plugins/multiproject/groovy", files="settings.grad
 
 You can also encapsulate the versions of external plugins by composing the build logic using your own <<sharing_build_logic_between_subprojects#sec:sharing_logic_via_convention_plugins,convention plugins>>.
 
+[[sec:root_plugins_dsl]]
+=== 3. Applying plugins from the project root
```

Comment:
```suggestion
=== 3. Applying plugins declared in the root project
```

I think it's important to be consistent with the terminology, though, I am not sure whether this terminology is the right one to use.

We can use "declared plugin" in the context of this page/section as the plugin that was apply-falsed. Maybe we should find a better word than "declared", but importantly it's not the same as "applied". I see that some other places in the text refer to this as "made available".

"Project root" is not a thing, or not a thing in this context, it should be "root project".

But also consider making this section more generic, talking about any parent project. Because the plugins don't have to be declared in the root project to be applied in the childred.

Reply:
Fixed. Thank you for the suggestion.